File: Day 45/homework/hw2.py
# 2) შექმენით ფუნქცია check_age, რომელიც არგუმენტად მიიღებს მომხმარებლის ასაკს.
# თუ მომხმარებლის ასაკი მეტი ან ტოლი იქნება 18-ზე, ტერმინალში დაიბეჭდოს "Access Granted", წინააღმდეგ შემთხვევაში – "Access Denied".

def check_age (age):
    if age >= 18:
        return("Access Granted")
    else:
        return ("Access Denied")
    
print(check_age(12))

File: Day 124/homework/hw2.js
// 2) შექმენით Arrow ფუნქცია, რომელიც მიიღებს მომხმარებლის ასაკს.
// თუ ასაკი არის 18 ან მეტი, დაბეჭდოს ტექსტი: "You're an adult"
// სხვა შემთხვევაში დაბეჭდოს ტექსტი: "You are not an adult".



// Traditional way
const checkAge  = (age) => {
    if(age >= 18){
        return 'You are an Adult'
    }else{
        return "You are not an adult"
    }
}

// Ternary Operator

const checkAge  = (age) => {
    return age >= 18 ? 'You are an Adult' : "You are not an Adult"
}